UnitySDK/Assets/ML-Agents/Scripts/Grpc/RpcCommunicator.cs

```diff
@@ -48,6 +50,8 @@ public class RpcCommunicator : ICommunicator
         /// The communicator parameters sent at construction
         CommunicatorInitParameters m_CommunicatorInitParameters;
 
+        Dictionary<int, SideChannel> m_SideChannelsDict = new Dictionary<int, SideChannel>();
```

Comment:
nit: maybe just `m_SideChannels`? `Dict` in the name feels redundant. not a big deal though.